src/components/internals/create-select-styles.js

```diff
@@ -157,11 +157,13 @@ const multiValueRemoveStyles = () => base => ({
   },
 });
 
-const groupStyles = () => base => ({
+const groupStyles = props => base => ({
   ...base,
   padding: 0,
-  '&:not(:firstChild)': {
-    borderTop: `1px ${vars.borderColorSeparator} solid`,
+  '&:not(:first-child)': {
+    borderTop: props.showOptionGroupDivider
+      ? `1px solid ${vars.colorGray}`
+      : base.borderTop,
```

Comment:
Is this correct? Where does it come from? 🤔

Reply:
base.borderTop is `undefined`. I think this is what we want?

Reply:
This value is undefined , but used `base.borderTop` instead of hardcoding `undefined` to stick to the patterns found elsewhere in this file

Reply:
This seems closed over from [here](https://github.com/commercetools/ui-kit/blob/47d5de1e53f5dca1a569bcd8f1158bbe10ace967/src/components/internals/create-select-styles.js#L195) with the `props` [here](https://github.com/commercetools/ui-kit/blob/7c53c7d3b301a3930917c8456eea2f87d7c446e0/src/components/inputs/select-input/select-input.js#L175). Then I fail to follow from react-select what `base is. Should that be our `vars`?

Reply:
From when i looked into it, `base` are the default styles for the component provided by `react-select` itself.

Reply:
Yeah, I think it's fine to use base.borderTop. We are doing similar things in the rest of these functions.